README.md

```diff
@@ -1,18 +1,115 @@
+## Transcoding video on Windows, the Don Melton way
 
+Don Melton provides some [amazing scripts](https://github.com/donmelton/video_transcoding) that wrap [HandBrake](https://handbrake.fr/) to produce high quality, portable versions of Blu Ray rips. The tools Don provides are written in [Ruby](https://www.ruby-lang.org/en/), and they're packaged as a [Ruby Gem](https://rubygems.org/). All the tools are cross platform, requiring only the Ruby runtime to be installed. But they do depend on external programs to get anything useful done. These dependencies are cross platform as well, but Windows has no standard package management system (de facto or otherwise), so installing these dependencies for Windows can be a bit of a pain. But it is possible! There are several ways to go about installing these tools and their dependencies. Downloading binaries manually, or using [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-gb/commandline/wsl/about) are two such options
 
-## The two methods:
-- [manually installing the binaries](https://github.com/JMoVS/installing_video_transcoding_on_windows/blob/master/native_method.md)
-- [installing it in the linux subsystem for windows](https://github.com/JMoVS/installing_video_transcoding_on_windows/blob/master/lxss_method.md)
+### Windows 10
+If you're using an up-to-date version of Windows 10, you have access to "[Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-gb/commandline/wsl/about)". This feature allows you to run command line Linux programs on Windows without using a virtual machine, and it also gives us access to Ubuntu's extensive package archives. Everything we need to run these scripts is easily installable with apt-get.
 
-## Notes about the two methods:
+1. First up, you need to activate Bash.
+    1. Open Settings then go to `Update & Security -> For Developers`
+    1. Change your computer to "Developer Mode"
+    1. Click `Start` then look for `Turn Windows features on or off`
+    1. Select `Windows Subsystem for Linux (beta)`
+    1. Click ok. You'll need to restart your PC
+    1. Now open up `cmd.exe` and run `bash`. Follow the instructions then bash is set up and ready to go 
+1. Next we install the dependencies for `video_transcoding`
+    1. Fire up bash
+    1. Run `sudo apt-get update && sudo apt-get upgrade` to ensure everything's up to date
```

Comment:
Also, the first time you run bash, it asks you to create a user and a password, should we mention that here as well?